projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

```diff
@@ -147,6 +95,7 @@ const Index = ( { criticalCss }: IndexProps ) => {
 						<span className="jb-badge">Upgraded</span>
 					</>
 				}
+				onEnable={ () => requestRegenerateCriticalCss() }
```

Comment:
Can this be just this too?

```suggestion
				onEnable={ requestRegenerateCriticalCss }
```

Reply:
Aye, I can do that. d326e2b8d7